README.md

```diff
@@ -3,6 +3,10 @@ This is a little Django web application that does an OpenDrift Leeway simulation
 
 This is an experimental tool to help Search and Rescue operations.
 
+Sample output from the leeway tool (100 points, 1km radius, near Lampedusa):
+
+![Example leeway output](./leeway-example-output.png)
```

Comment:
Do you think it would make sense to move this image into a subdirectory, e.g. `.github`?